Lib/test/test_posix.py

```diff
@@ -283,6 +307,34 @@ def test_pwrite(self):
         finally:
             os.close(fd)
 
+    @unittest.skipUnless(hasattr(posix, 'pwritev'), "test needs posix.pwritev()")
+    def test_pwritev(self):
+        fd = os.open(support.TESTFN, os.O_RDWR | os.O_CREAT)
+        try:
+            n = os.pwritev(fd, [b'test1tt2t3'],2)
+            self.assertEqual(n, 10)
+
+            os.lseek(fd, 0, os.SEEK_SET)
+            self.assertEqual(b'\x00\x00test1tt2', posix.read(fd, 10))
+
+        finally:
+            os.close(fd)
+
+
+    @unittest.skipUnless(hasattr(posix, 'os.RWF_SYNC'), "test needs posix.pwritev2()")
+    def test_pwritev2(self):
```

Comment:
ditto, rename to test_pwritev_flags() and change the skip message to not mention a function that doesn't exist.